docs/core/install/linux-package-manager-ubuntu-1910.md

```diff
@@ -95,7 +95,7 @@ If that doesn't work, you can run a manual install with the following commands.
 
 ```bash
 sudo apt-get install -y gpg
-wget O- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o microsoft.asc.gpg
+wget -O- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o microsoft.asc.gpg
```

Comment:
I'd expect this to conflict with https://github.com/dotnet/docs/pull/18141, which fixes it with slightly different (?) syntax. This PR is more consistent with the other pages, though.